examples/wallet/app/actions/router.js

```diff
@@ -3,10 +3,31 @@ import { push } from 'connected-react-router';
 import type { Dispatch, GetState } from '../reducers/types';
 import type { Address } from '../models';
 import routes from '../constants/routes';
+import {
+  updateAccountTransactionsAndState,
+  ACCOUNT_STATE_ERROR
+} from './account';
+import { readAccountKeysFromLocalStorage } from '../utils/storage';
+
+export const SET_KEYS = 'SET_KEYS';
 
 // eslint-disable-next-line import/prefer-default-export
 export const redirectToFirstAppPage = () => {
   return (dispatch: Dispatch, getState: GetState) => {
+    const accountKeys = readAccountKeysFromLocalStorage();
+    if (accountKeys) {
+      dispatch({
+        type: SET_KEYS,
+        ...accountKeys
+      });
+      return dispatch(updateAccountTransactionsAndState()).catch(error => {
+        if (error.message === ACCOUNT_STATE_ERROR) {
+          return dispatch(push(routes.WALLET));
```

Comment:
The execution doesn't enter here when the node is down, and the user is redirected to the CHOOSE_RESTORE_OR_IMPORT screen.